### PR TITLE
Adds README

### PR DESCRIPTION
**Pull request by ignasi35**
Better than nothing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Omnidoc
+Omnidoc is an sbt script piggy-backing the `publish` task that aggregates all source code and manuals produced across the Play ecosystem and produces a single deliverable with:
```
Comment (dwijnand) [inline README.md]:
```suggestion

Omnidoc is an sbt build that adds to sbt's `mappings in (Compile, packageBin)` to aggregate source code and manuals produced within the Play ecosystem and produce a single deliverable which includes:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ * manual (also referred to as `playdoc`)
+ * javadoc
+ * scaladoc
+
+## How it works
+
+Omnidoc defines [5 tasks]([omnidoc/OmnidocBuild.scala at 70f04533d0f881a9a7f6c1ac5ec6af1d8bb335f9 · playframework/omnidoc · GitHub](https://github.com/playframework/omnidoc/blob/70f04533d0f881a9a7f6c1ac5ec6af1d8bb335f9/project/OmnidocBuild.scala#L88-L92)) to:
```
Comment (dwijnand) [inline README.md]:
```suggestion
Omnidoc defines [5 tasks](https://github.com/playframework/omnidoc/blob/70f04533d0f881a9a7f6c1ac5ec6af1d8bb335f9/project/OmnidocBuild.scala#L88-L92) to:
```

**Comment by ignasi35** [comment]
thanks @dwijnand

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Omnidoc
+
+Omnidoc is an sbt build that adds to sbt's `mappings in (Compile, packageBin)` to aggregate source code and manuals produced within the Play ecosystem and produce a single deliverable which includes:
```
Comment (ignasi35) [inline README.md]:
```suggestion
Omnidoc is an sbt build that adds to sbt's `mappings in (Compile, packageBin)` to aggregate source code and manuals produced within the Play ecosystem and produce a single deliverable. This must not be confused with Interplay's [Omnidoc](https://github.com/playframework/interplay/blob/master/src/main/scala/interplay/Omnidoc.scala) which _simply_ adds some metadata on the `-source.jar` artifact of every Play library. 

The resulting deliverable includes:
```

**Pull request merged**